src/index.tsx

```diff
@@ -39,12 +39,14 @@ interface BuiltInValidationAttrs {
 // Valid attributes by input type.  See:
 // https://html.spec.whatwg.org/multipage/input.html#do-not-apply
 
-type InputTextValidationAttrs = {
-  type?: "text" | "search" | "url" | "tel" | "email" | "password";
-} & Pick<
-  BuiltInValidationAttrs,
-  "required" | "minLength" | "maxLength" | "pattern"
->;
+type InputTextValidationAttrs = (
+  | { type?: "text" | "search" | "url" | "tel" | "password" }
+  | { type: "email"; multiple?: boolean }
+) &
```

Comment:
Added support for `multiple` attribute of `email` input

---

```html
<input type="email" name="a" />
<input type="email" name="a" />
<input type="email" name="a" />
```
```ts
inputs: {
  a: {
    multiple: true
    validationAttrs: {
      type: "email",
    }
  }
}
```

---

```html
<input type="email" name="a" multiple />
```
```ts
inputs: {
  a: {
    validationAttrs: {
      type: "email",
      multiple: true
    }
  }
}
```

---

```html
<input type="email" name="a" />
<input type="email" name="a" multiple />
```
```ts
inputs: {
  a: {
    multiple: true
    validationAttrs: {
      type: "email",
      multiple: true
    }
  }
}
```